frontend/src/metabase/palette/constants.ts

```diff
@@ -19,3 +21,74 @@ export const GROUP_LABELS = {
 };
 
 export const ELLIPSIS = "...";
+
+type AdminSettingsSections = Record<
+  string,
+  {
+    name: string;
+    hidden?: boolean;
+    adminOnly?: boolean;
+  }
+>;
+
+export const getAdminSettingsSections = (
+  settings: Settings,
+): AdminSettingsSections => ({
```

Comment:
This is the former `ADMIN_SETTINGS_SECTIONS` object, with the unnecessary stuff stripped out, and the format changed slightly. Putting it in a function, rather than a static object also fixes a localization issue.